### PR TITLE
Fixed accent color selection when no yellow is shown

### PR DESCRIPTION
**Pull request by mikeger**
# Issue

The indexes where not aligned between `ZMAccentColor.all()` array and `self.colors` array.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Settings/AccentColorPickerController.swift [hunk 1]
```diff
@@ -212,13 +212,17 @@ extension ColorPickerController: UITableViewDelegate, UITableViewDataSource {
...
 
 
-open class AccentColorPickerController: ColorPickerController {
+final class AccentColorPickerController: ColorPickerController {
+    fileprivate let allAccentColors: [ZMAccentColor]
+    
+    
     public init() {
-        let allColors = ZMAccentColor.all().filter { $0 != .brightYellow }
+        self.allAccentColors = ZMAccentColor.all().filter { $0 != .brightYellow }
```
Comment (marcoconti83) [inline Wire-iOS/Sources/UserInterface/Settings/AccentColorPickerController.swift]: Feels a bit hacky to hardcode one out. But from what I understood we need to keep this yellow around for older clients, right?

Comment (marcoconti83) [inline Wire-iOS/Sources/UserInterface/Settings/AccentColorPickerController.swift]: Maybe store it in a static `discontinuedColors` array makes it clearer why we are leaving out yellow

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Settings/AccentColorPickerController.swift]: Yep, if user used to have yellow color we are not forcing it to be changed (yet)

**Comment by mikeger** [comment]
@marcoconti83 please re-review

**Comment by marcoconti83** [comment]
👍

**Pull request merged**